Sources/Cells/UserAdTableViewCell/UserAdTableViewCell.swift

```diff
@@ -0,0 +1,224 @@
+//
+//  Copyright © 2020 FINN AS. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+public class UserAdTableViewCell: UITableViewCell {
+
+    // MARK: - Public properties
+
+    public enum Style {
+        case `default`
+        case compressed
+    }
+
+    public var remoteImageViewDataSource: RemoteImageViewDataSource? {
+        didSet {
+            adImageView.dataSource = remoteImageViewDataSource
+        }
+    }
+
+    public var loadingColor: UIColor? = .toothPaste
+
+    // MARK: - Private properties
+
+    private var model: UserAdTableViewCellViewModel?
+
+    private var style: Style? = .default {
```

Comment:
Have you considered just getting rid of this property(?) The code handling it is very close to a pure functional form already. ☺️